cmake/CMakeHelper.cmake

```diff
@@ -92,6 +92,7 @@ macro(COLMAP_ADD_LIBRARY)
     target_link_libraries(${COLMAP_ADD_LIBRARY_NAME}
         PRIVATE ${COLMAP_ADD_LIBRARY_PRIVATE_LINK_LIBS}
         PUBLIC ${COLMAP_ADD_LIBRARY_PUBLIC_LINK_LIBS})
+    target_compile_definitions(${COLMAP_ADD_LIBRARY_NAME} PUBLIC ${COLMAP_COMPILE_DEFINITIONS})
```

Comment:
We'll also need to add this to the COLMAP_ADD_EXECUTABLE and COLMAP_ADD_TEST macros.

Reply:
There could be executables that don't depend on libraries (and I guess in theory also tests, but that practically doesn't make sense). It's safer to just add them in both cases.

Reply:
Thanks for clarifying! I just wanted to make sure. I will add the changes.